webapp/src/index.tsx

```diff
@@ -460,9 +461,20 @@ export default class Plugin {
                     return;
                 }
 
+                const iceConfigs = [...iceServers(store.getState())];
+                if (needsTURNCredentials(store.getState())) {
+                    logDebug('turn credentials needed');
+                    try {
+                        const resp = await axios.get(`${getPluginPath()}/turn-credentials`);
```

Comment:
Just a note: we're going to have to replace these with Client4 requests. Not sure if you want to do that here, or wait for the ticket to do it bulk. Up to you!

Reply:
I like `axios` :) I think we can defer to the ticket mostly because it's a bit more involved than just replacing a call. I'd like to make sure to have a proper object/superclass or something tidy. I like the approach we have on mobile, not sure if easily replicable here though.